airflow/jobs.py

```diff
@@ -544,7 +544,6 @@ def __init__(
             subdir=settings.DAGS_FOLDER,
             num_runs=-1,
             processor_poll_interval=1.0,
-            run_duration=None,
```

Comment:
Is this a part of this PR? This is already a part of https://github.com/apache/incubator-airflow/pull/4320

Reply:
Auch, good catch